carbonnelleg/LELEC210X_GROUP_E
# Integration gr310 (#42)

* chore: simplify measurements method

* [pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

* tests with logging

* [pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

* Add tx power indicator to GNU GUI and log

* oops

* chore(tex): add note about timing

* :-(

* Correct the logger since Jerome cannot code + DC offset blocker

* update logger config

* fix?

* fixes and better logging

* more work bc i am so smart

* Update noise estimation to account for dc offset anyway

* better format payload

* Update logger

* parse measurements

* fix tex

* Add on Query noise estimation

* Query for Noise estimation on push button is ok and linked with sync.

* Functional main_app

* Support for GNU Radio 3.10 added

* Add GR310 files

* Enable support of 3.8 and 3.10 on same branch

* Correct some files

* Update GNU Radio GRC for H3b

* Update INSTALL.md for GR310 support

* Update INSTALL.md

* Update INSTALL.md

* Update README.md

* Update hands-on 3b

* Remove demod parts

* Finalize H3b

* [pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

* Update pyproject.toml

* [pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

---------
